frontend/src/shared/helpers/clear-empty-objects.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Recursively delete any empty object members
+ * @param o target object
+ */
+export const clearEmptyObjects = (o: any) => {
```

Comment:
nice 👍 what did you use this for? seems like it's not used in the pr 🤔

Reply:
it's been used inside the onSubmit function, it's been used to clean empty filters in the `ReportGetDTO` payload before requesting a `POST` request for the report export endpoint 😁

![image](https://user-images.githubusercontent.com/90462635/193790987-d324828a-f1fd-4346-b0ef-7f68fe1a6e68.png)